public/controllers/management/components/management/configuration/utils/wz-fetch.js

```diff
@@ -379,7 +379,7 @@ export const saveConfiguration = async (selectedNode, xml) => {
       await saveFileManager(xml);
     }
   } catch (error) {
-    return Promise.reject(error.message || error);
+    throw error;
```

Comment:
**thought**: I am not sure if removing the throw of `error.message` could be a problem. I reviewed and it seems the `saveConfiguration` function is not used. Maybe we could remove it.

Reply:
Thanks @Desvelao. I removed `saveConfiguration` from `wz-fetch` file